src/navbar/brand.tsx

```diff
@@ -30,6 +30,14 @@ export const Brand = ({
             onClick(event, type);
         }
     };
+
+    const handleError = (
+        event: React.SyntheticEvent<HTMLImageElement, Event>
+    ) => {
+        (event.target as HTMLImageElement).onerror = null;
+        (event.target as HTMLImageElement).src = require("./img/default.png");
```

Comment:
I'm getting a `Module not found: Can't resolve './img/default.png'` error when loading the component. I think the import needs to be at the top level for rollup to bundle it into the build

Reply:
looking at the [rollup docs](https://www.npmjs.com/package/@rollup/plugin-image), you should be able to do `import image from './img/default.png';`